packages/rs-dapi-client/src/dapi_client.rs

```diff
@@ -194,81 +251,40 @@ impl DapiRequestExecutor for DapiClient {
                     address: Some(address.clone()),
                 })?;
 
-                let response = transport_request
+                let result = transport_request
                     .execute_transport(&mut transport_client, &applied_settings)
                     .await
                     .map_err(DapiClientError::Transport);
 
-                match &response {
-                    Ok(_) => {
-                        // Unban the address if it was banned and node responded successfully this time
-                        if address.is_banned() {
-                            let mut address_list = self
-                                .address_list
-                                .write()
-                                .expect("can't get address list for write");
-
-                            address_list.unban_address(&address).map_err(|error| {
-                                ExecutionError {
-                                    inner: DapiClientError::AddressList(error),
-                                    retries: retries_counter
-                                        .load(std::sync::atomic::Ordering::Acquire),
-                                    address: Some(address.clone()),
-                                }
-                            })?;
+                let retries = retries_counter.load(std::sync::atomic::Ordering::Acquire);
```

Comment:
This operation will execute in one thread anyway, so maybe we can use `Relaxed` here?

Reply:
Tokio runtime is using a pool of threads to execute tasks so they might be in different threads, ins't?

Reply:
Yes, but my understanding is that this code is already synchronized, that is, this code will not execute concurrently in different threads. So no need for additional synchronization.

Is it correct?

Reply:
How is it synchronized? If you call an async function that it's a future that can be executed on any thread in async runtime.

Reply:
My point is that I don't expect it to be executed in parallel by the retry logic. 

This routine is started by backon::retry logic (line 293+) which will execute it, then it will  check the result, then it will execute second time, etc. There should be no case where access to retries_counter is called in two threads simultaneously. 

But we can keep Acquire if you see some reason for that, it shouldn't have a big impact.

Reply:
Yeah, maybe you right. Ok, I will change to Relaxed